features/cli/benchmark_grading_cases.feature.md

```
@@ -0,0 +1,49 @@
+        tolerance: 1e-9
+        items:
+          - type: run
+            expected:
+              - basis: "|1>"
+                amplitude:
+                  real: 1
+                  imaginary: 0
+  ---
+
+  checks と grading_cases を同時指定した不正な課題です。
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)